pkg/resolution/resolver/http/params.go

```diff
@@ -14,6 +14,15 @@ limitations under the License.
 package http
 
 const (
-	// urlParam is the url to fetch the task from
-	urlParam string = "url"
+	// urlParamType is the URL to fetch the task from
+	urlParamType string = "url"
+
+	// httpBasicAuthUsername is the user name to use for basic auth
+	httpBasicAuthUsername string = "http-username"
+
+	// httpBasicAuthSecret is the reference to a secret in the PipelineRun namespace to use for basic auth
```

Comment:
This is not only PipelineRun, but could be TaskRun?

Reply:
i have the adjusted the comment